src/common.js

```diff
@@ -79,7 +79,7 @@ function quotepath(file) {
 function stdoutFilter(query) {
   return process.platform == "win32"
     ? ' | findstr /v "' + query + '"'
-    : ' | grep -v "' + query + '"';
+    : ' | ( grep -v "' + query + '" || true )';
```

Comment:
when will it fail to grep?

Reply:
when there's no match, code will be 1. There's a flag to supress errors (code 2), but not for this. Stupid, but needed for now.